rcv/frame_handlers/frc_2019.py

```diff
@@ -26,6 +26,8 @@ def __init__(self, configuration):
         self._points = None
         self._line_contour = None
         self._response = None
+        self._is_front = self._conf.getboolean('rcv_server', 'is_front')
```

Comment:
This is a property of the camera and not of the frame, there for this should be handled in the nes56_cv_manager module.

Reply:
So what i should do is on every sending from nes56_cv_manager add the front ot back field to the sent dictionary?